docs/components/empty-state.md

```diff
@@ -223,6 +280,30 @@ Slot for passing title text.
 </KEmptyState>
 ```
 
+### supportingText
+
+Slot for a secondary message or additional information (e.g. pricing).
+
+<KEmptyState
+  message="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla nec purus feugiat, molestie ipsum et, consequat nibh."
+  title="Empty State With Supporting Text"
+>
+  <template #supporting-text>
+    <b>Pricing:</b> starting at $30 per user per month.
+  </template>
+</KEmptyState>
+
+```html
+<KEmptyState
+  message="Lorem ipsum dolor sit amet..."
+  title="Empty State With Supporting Text"
+>
+  <template #supporting-text>
+    <b>Pricing:</b> starting at $30 per user per month.
+  </template>
+</KEmptyState>
+```
```

Comment:
question: it seems like this could just be part of the default slot. Why does it need to be separate?

nit: if we are going to keep it, would a slot name of `footer` be more appropriate?